lib/validators/liveValidator.ts

```diff
@@ -332,29 +350,132 @@ export class LiveValidator {
     return result
   }
 
+  /**
+   *  Validates live request.
+   */
+  public validateLiveRequest(
+    liveRequest: LiveRequest,
+    specOperation?: Operation
+  ): RequestValidationResult {
+    let operation = specOperation
+    if (!operation) {
```

Comment:
Is there a use case for validating the request against a different operation than the one specified by the request url + method? If not we may want to remove specOperation from the parameters.

Reply:
well, that part was introduced mostly so that if we already have the operation, we dont need to find it again ( see usage in `validateRequestResponse`)

Reply:
Couple questions then:
- Do we return the operation so it can be passed in again? Looks like we return an OperationInfo which has an OperationId but not an Operation... 
- Should we check for cases where the provided Operation doesn't match what we would have resolved if it were not provided? Maybe this is an error/warning condition?

Reply:
> Do we return the operation so it can be passed in again? Looks like we return an OperationInfo which has an OperationId but not an Operation...
We dont want to return the operation, we just return the validation result ( which will contain the op_id against it was executed ). usually you would need like you mentioned only the method and url, thats why we don't expose the `findSpecOperation` since except internally doesnt really makes sense to have it ( unless you test some stuff and you keep the url and method I guess and just change the payload, for that case we could expose it ).

> Should we check for cases where the provided Operation doesn't match what we would have resolved if it were not provided? Maybe this is an error/warning condition?
Wellll, i mean in that case we can even skip passing it, since the point is to skip having to do that.